src/libraries/System.Private.CoreLib/src/System/Globalization/CalendarData.Unix.cs

```diff
@@ -12,5 +14,14 @@ internal sealed partial class CalendarData
         internal static int GetCalendarsCore(string localeName, bool useUserOverride, CalendarId[] calendars) =>
             IcuGetCalendars(localeName, calendars);
 #pragma warning restore IDE0060
-     }
+
+        internal static int GetTwoDigitYearMax(CalendarId calendarId)
+        {
+            Debug.Assert(!GlobalizationMode.UseNls);
+
+            // There is no user override for this value on Linux or in ICU.
+            // So just return -1 to use the hard-coded defaults.
+            return -1;
```

Comment:
>return -1; [](http://example.com/codeflow?start=12&length=10)

Could you please change this to:

```C#
return GlobalizationMode.Invariant ? Invariant.iTwoDigitYearMax : -1;
```

Reply:
Sure thing, this has been corrected

Reply:
please check there's a test that reads this in invariant mode (ie that would fail without the change above)